src/Microsoft.ML.Data/Data/SchemaDefinition.cs

```diff
@@ -68,30 +68,95 @@ public VectorTypeAttribute(params int[] dims)
     /// column encapsulates.
     /// </summary>
     [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
-    public sealed class ColumnAttribute : Attribute
+    public sealed class LoadColumnAttribute : Attribute
     {
-        public ColumnAttribute(string ordinal, string name = null)
+        /// <summary>
+        /// Initializes new instance of <see cref="LoadColumnAttribute"/>.
+        /// </summary>
+        /// <param name="ordinal">The index of the column in the text file.</param>
+        /// <param name="name">The optional name of the column, if it should be different from the name of the field or property where the attribute is positioned.</param>
+        /// <param name="loadAllOthers">Wheather to load all columns, besides the one specified in the ordinal.</param>
+        public LoadColumnAttribute(int ordinal, string name = null, bool loadAllOthers = false)
```

Comment:
Do you think this will be useful when the user wants to load all features into one column, and the label in another. 

we'd spare them having to look up their cols cardinality, and a concat. 

---
In reply to: [242673339](https://github.com/dotnet/machinelearning/pull/1878#discussion_r242673339) [](ancestors = 242673339)

Reply:
still not enough reason, in my opinion

---
In reply to: [242741558](https://github.com/dotnet/machinelearning/pull/1878#discussion_r242741558) [](ancestors = 242741558,242673339)

Reply:
removed

---
In reply to: [243121936](https://github.com/dotnet/machinelearning/pull/1878#discussion_r243121936) [](ancestors = 243121936,242741558,242673339)